string robot_name
bool record
bool watchdog 

---
bool success
string error